apps/deploy-web/tests/ui/fixture/context-with-extension.ts

```diff
@@ -49,9 +50,29 @@ export const test = baseTest.extend<{
       // ignore timeout error
     }
 
-    const extPage = await getExtensionPage(context, extensionId);
```

Comment:
it is actually a good idea to have that method getExtensionPage. I get it probably was added while you were working on this PR, but it's already in main and it doesn't make sense to remove it.